components/dropdownlist/filter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Filter
+page_title: DropDownList - Filter
+description: Filtering in the DropDownList for Blazor.
+slug: components/dropdownlist/filter
+tags: telerik,blazor,drop,down,list,dropdownlist,filter
+published: True
+position: 5
+---
+
+# DropDownList Filter
+
+The Filter in the DropDownList component is an input shown at the top of the dropdown popup, used for filtering the underlying datasource.
+
+To enable filtering, set the `Filterable` parameter to `true`.
+
+Filtering ignores casing and the default filter operator is `starts with`. Filtering looks in the `TextField`, and the filter is reset when the dropdown closes. You can choose a different operator through the `FilterOperator` parameter that takes a member of the `Telerik.Blazor.StringFilterOperator` enum.
+
+You can also implement custom (server) filtering and set a data source dynamically through the [`OnRead` event]({%slug components/dropdownlist/events%}#onread).
+
+>caption Filtering in the DropDownList
+
+````CSHTML
+@* Type something in the input to see items whose text contains only the typed string, for example "uct 2" *@
```

Comment:
Maybe tweak this a little to denote that you should first open the dropdown. I personally find that UX a tad strange, so it may be a nice idea to record a gif of the component behavior and include it.

Reply:
Done.